### ISSUE TITLE
[advanced] Make data model more explicit and reasonable

### ISSUE THREAD

**Issue by besteady** [Volodimirich/MadeFinalProject#5]
Currently its:
```python
class Paper(BaseModel):
    id: Optional[PydanticObjectId] = Field(None, alias="_id")
    title: str
    authors: Optional[List[Dict[str, Any]]]
    venue: Dict[str, Any]
    year: int
    keywords: List[str]
    n_citation: Optional[int]
    page_start: Optional[str]
    page_end: Optional[str]
    lang: Optional[str]
    volume: Optional[str]
    issue: Optional[str]
    issn: Optional[str]
    isbn: Optional[str]
    doi: Optional[str]
    pdf: Optional[str]
    url: Optional[List[str]]
    abstract: Optional[str]
```

We should check what are fields types and can they be missed in our dataset

**Comment by koluzaeva** [comment]
If it is convenient, then we can store all the fields and issue them as a response to the user's request, but according to the project, much is not needed and will not be used in the part of the DS
 
authors: Optional[List[Dict[str, Any]]]   --> [{
'_id': '53f395d0dabfae4b34a71643',  --> str 
 'name': 'Laura Griffiths',   --> str
'org': 'Mirman School',  --> str  (can be deleted if we will not use it in "author-to-author recomendations")
 'gid': '5b869d7fe1cd8e14a3a2dc35' --> str  (gid - "grid Id for affiliation" ??? not sure,  I think it is not needed)
}],

'venue': Dict[str, Any] -->
{
'raw': 'SIGGRAPH Abstracts and Applications', --> str
  'raw_zh': None, -->  delete
  'publisher': None, --> str (it can be useful in search by venue name as two venues might have the same name and different publisher)
  'type': 10 --> int  (not useful) 
},

    year: int (required )
    keywords: List[str]  (required)
    n_citation: Optional[int] (required )
    page_start: Optional[str] (not useful) 
    page_end: Optional[str]  (not useful) 
    lang: Optional[str] (required   en :4725974,  zh:389, missing: 627946)
    volume: Optional[str]  (not useful) 
    issue: Optional[str]  (not useful) 
    issn: Optional[str] (not useful) 
    isbn: Optional[str] (not useful) 
    doi: Optional[str] (required )
    pdf: Optional[str] (it will be useful for representation of the search results )
    url: Optional[List[str]] ( we can use doi and pdf to find paper link, but doi is available only for 3994566 items )
    abstract: Optional[str] (required )

**Comment by besteady** [comment]
Closed with #13
